<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CProductInstanceInfoEditorComp</class>
 <widget class="QWidget" name="CProductInstanceInfoEditorComp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Product Instance</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="InstanceIdGroup">
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Instance-ID</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="ProductInstanceIdEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="CustomerGroup">
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Customer</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QComboBox" name="CustomerCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ProductSelectionGroup">
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Product</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="ProductCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="LicensesGroup">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>104</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>License</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QComboBox" name="LicenseCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ExpireGroup">
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777200</height>
      </size>
     </property>
     <property name="title">
      <string>Valid Until</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QDateTimeEdit" name="ValidUntilDate">
        <property name="date">
         <date>
          <year>2022</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
        <property name="currentSectionIndex">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../imtgui/imtgui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ExpireGroup</sender>
   <signal>toggled(bool)</signal>
   <receiver>ValidUntilDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
